components/pdf4me/common/utils.mjs

```
@@ -0,0 +1,46 @@
+  const filePath = normalizeFilePath(filename);
+  fs.writeFileSync(filePath, buffer);
+
+  return [
+    filename,
+    filePath,
+  ];
+}
+
+function handleErrorMessage(error) {
+  let errorMessage = error.name;
+  if (error.response && error.response.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)